### PR TITLE
Fix deletion of ACM backup resources and support optional args

### PR DESCRIPTION
**Pull request by philipwu08**

### CODE REVIEW

File: dr-script/migrate-hcp.sh [hunk 1]
```diff
@@ -316,6 +314,9 @@ function render_svc_objects() {
 
 function backup_hc() {
     BACKUP_DIR=${HC_CLUSTER_DIR}/backup
+                
+    rm -f -r ${BACKUP_DIR}
```
Comment (mikeshng) [inline dr-script/migrate-hcp.sh:318]: Let's error exit out if the folder exist and ask the user to move/delete the existing backup folder manually?

File: dr-script/migrate-hcp.sh [hunk 2]
```diff
@@ -478,14 +479,52 @@ function teardown_old_svc() {
...
 if [ -z $HCPASS ]; then
+    echo "No value for HCPASS parameter specified"
     exit 1
 fi
 
-source $REPODIR/dr-script/common.sh
+HC_CLUSTER_DIR="${BASE_PATH}/${HC_CLUSTER_NAME}"
+HC_KUBECONFIG="${HC_CLUSTER_DIR}/kubeconfig"
+BACKUP_DIR=${HC_CLUSTER_DIR}/backup
+HASH=${HC_CLUSTER_NAME:7:4}
```
Comment (mikeshng) [inline dr-script/migrate-hcp.sh:527]:
```suggestion
```
I don't think HASH is used anywhere.

File: dr-script/migrate-hcp.sh [hunk 3]
```diff
@@ -495,7 +534,8 @@ echo "Backup Done!"
 ELAPSED="Elapsed: $(($SECONDS / 3600))hrs $((($SECONDS / 60) % 60))min $(($SECONDS % 60))sec"
 echo $ELAPSED
 
-## Migration
+# Migration
+
```
Comment (mikeshng) [inline dr-script/migrate-hcp.sh:538]: lets undo this change so there is less line changes when we finally create a PR against the SRE team

File: dr-script/migrate-hcp.sh [hunk 4]
```diff
@@ -504,7 +544,8 @@ echo "Restoration Done!"
 ELAPSED="Elapsed: $(($SECONDS / 3600))hrs $((($SECONDS / 60) % 60))min $(($SECONDS % 60))sec"
 echo $ELAPSED
 
-## Teardown
+# Teardown
+
```
Comment (mikeshng) [inline dr-script/migrate-hcp.sh:548]: lets undo this change so there is less line changes when we finally create a PR against the SRE team

File: dr-script/common.sh [hunk 5]
```diff
@@ -32,9 +32,7 @@ HC_CLUSTER_ID=""
 HC_CLUSTER_NS=""
 HC_REGION=""
 HC_CLUSTER_NAME=""
-HC_CLUSTER_DIR="${BASE_PATH}/${HC_CLUSTER_NAME}"
-HC_KUBECONFIG="${HC_CLUSTER_DIR}/kubeconfig"
-BACKUP_DIR=${HC_CLUSTER_DIR}/backup
+HCPASS=""
```
Comment (mikeshng) [inline dr-script/common.sh:35]:
```suggestion
HC_PASS=""
```
I think we should update the HCPASS to HC_PASS just the same as other `HC_` vars. 
I should've done it in the last PR. My bad.

File: dr-script/migrate-hcp.sh (hunk 1 above)
Comment (philipwu08) [reply dr-script/migrate-hcp.sh:318]: Agree, don't want to lose any data.

File: dr-script/migrate-hcp.sh (hunk 2 above)
Comment (philipwu08) [reply dr-script/migrate-hcp.sh:527]: I believe Hash is used in the upload of the etcd backup to the s3 bucket.

File: dr-script/common.sh (hunk 5 above)
Comment (philipwu08) [reply dr-script/common.sh:35]: Ok, let me do that for now, but I'm hoping to automate and not require this eventually.

File: dr-script/migrate-hcp.sh (hunk 2 above)
Comment (mikeshng) [reply dr-script/migrate-hcp.sh:527]: @philipwu08 I only saw `SIGNATURE_HASH` could you check please. Thanks.

Comment (philipwu08) [reply dr-script/migrate-hcp.sh:527]: You're right Mike. That's a different signature hash. I'll remove this.
